### PR TITLE
Kllsel create entity credit card for database

### PR DESCRIPTION
**Pull request by Kllsel**
CreditCard entity created and initialized

### CODE REVIEW

File: data_access/Entities/Configs/CreditCardConfig.cs [hunk 1]
```diff
@@ -14,9 +14,11 @@ public void Configure(EntityTypeBuilder<CreditCard> builder)
             builder.Property(x => x.Type).HasMaxLength(50);
             builder.HasIndex(x => x.Number).IsUnique();
             builder.HasIndex(x => x.VerificationCode).IsUnique();
-           
-
+            builder.Property(x => x.Validity).HasColumnType("date");
+            builder.Property(x => x.StartDate).HasColumnType("date");
```
Comment (OlexKov) [inline data_access/Entities/Configs/CreditCardConfig.cs]: Remove empty line

**Review by OlexKov — CHANGES_REQUESTED** [review]

**Comment by OlexKov** [comment]
You still need to add collections to the User and Category classes. One-to-many relationship. 
See an example Universal

File: data_access/Entities/CreditCard.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+{
+    public class CreditCard
+    {
+        public int Id { get; set; }
+        public int CategoryId { get; set; }
+        public string OwnerName { get; set; }
+        public string Type { get; set; }
+        public string Number { get; set; }
+        public string VerificationCode { get; set; }
+
```
Comment (OlexKov) [inline data_access/Entities/CreditCard.cs]: Add references to User and Category classes

**Review by OlexKov — CHANGES_REQUESTED** [review]

File: data_access/Entities/Configs/CreditCardConfig.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace data_access.Entities.Configs
+{
+    internal class CreditCardConfig : IEntityTypeConfiguration<CreditCard>
+    {
+        public void Configure(EntityTypeBuilder<CreditCard> builder)
+        {
+
```
Comment (OlexKov) [inline data_access/Entities/Configs/CreditCardConfig.cs]: Add OneToMany parameters

**Review by OlexKov — CHANGES_REQUESTED** [review]

**Review by Kllsel — COMMENTED** [review]
List changed to HasSet, added relations

**Review by OlexKov — APPROVED** [review]

**Comment by shaynmundane** [comment]
no errors have been noticed so far

**Review by shaynmundane — APPROVED** [review]
no errors have been noticed so far

**Pull request merged**